UnityProject/Assets/Scripts/PlayGroups/PlayerSprites.cs

```diff
@@ -17,12 +17,32 @@ public class PlayerSprites : NetworkBehaviour
 	public ClothingItem[] characterSprites; //For character customization
 	private CharacterSettings characterSettings;
 
+	private SpriteRenderer ghostRenderer; //For ghost sprites
+	private readonly Dictionary<Orientation, Sprite> ghostSprites = new Dictionary<Orientation, Sprite>();
+
 	private void Awake()
 	{
 		foreach (ClothingItem c in GetComponentsInChildren<ClothingItem>())
 		{
 			clothes[c.name] = c;
 		}
+
+		foreach (Sprite s in SpriteManager.PlayerSprites["mob"])
+		{
+			if (s.name == "mob_268")
+				ghostSprites.Add(Orientation.Down, s);
+
+			if (s.name == "mob_269")
+				ghostSprites.Add(Orientation.Up, s);
+
+			if (s.name == "mob_270")
+				ghostSprites.Add(Orientation.Right, s);
+
+			if (s.name == "mob_271")
+				ghostSprites.Add(Orientation.Left, s);
+		}

```

Comment:
Consider doing this instead to save on load up time if there is going to be a lot of players in the game:
```cs
ghostSprites.Add(Orientation.Down, SpriteManager.PlayerSprites["mob"][268]);
ghostSprites.Add(Orientation.Up, SpriteManager.PlayerSprites["mob"][269]);
ghostSprites.Add(Orientation.Right, SpriteManager.PlayerSprites["mob"][270]);
ghostSprites.Add(Orientation.Left, SpriteManager.PlayerSprites["mob"][271]);
```